wavefront/server/plugins/datasource/datasource/redshift/__init__.py

```
@@ -64,6 +64,19 @@ def fetch_data(
     def insert_rows_json(self, table_name: str, data):
         pass
 
+    async def update_rows_json(
+        self, table_name: str, filter: Dict[str, Any], data: Dict[str, Any]
```

Ruff (0.15.9)

[error] Function argument `filter` is shadowing a Python builtin

(A002)